ingest_power_lines.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import geojson
+import sys
+from elasticsearch import Elasticsearch                                                         # imports
+
+
+data_url = 'https://opendata.arcgis.com/datasets/70512b03fe994c6393107cc9946e5c22_0.geojson'    # sets up variables for future use
+if len(sys.argv) > 1:
+   data_url = sys.argv[1]                                                                       # option for user to choose their own data url
+index_name = 'powerlines'
+type = 'multilinestring'
+
```

Comment:
Two things.
For constants I like the codeing convention that they are all UPPERCASE words.
Second don't intermix code and constants.   Move the argv check down into the main body.

Reply:
I made a separate function that takes the user input. The only constant now is type, so that has been separated from the other variables.